tests/test_updater.py

```diff
@@ -1763,26 +1744,6 @@ def test_11__verify_metadata_file(self):
           metadata_file_object, 'root')
 
 
-  def test_12__get_file(self):
```

Comment:
Should we replace with a similar test for `_get_target_file()`? Is it tested somewhere else?

Reply:
`_get_target_file()` is exercised by the existing tests for `updater.download_target()` – not sure it warrants an extra test? `download_target()` only marshals the parameters for `_get_target_file()` and ensures that the directory we fetch into exists.

Reply:
A test for `_get_target_file()` would only be ~5LOC, so no great effort to add.

Reply:
As long as the function is covered somewhere, I have no strong objection.